Formula/fontforge.rb

```diff
@@ -16,71 +16,72 @@ class Fontforge < Formula
 
   deprecated_option "with-gif" => "with-giflib"
 
-  # Autotools are required to build from source in all releases.
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  if build.head?
```

Comment:
I think I originally had a `head do` block but rubocop complained because I also had an `if head?` further down in the compilation step.